localMatchApp/Headers/logIn.h

```diff
@@ -0,0 +1,18 @@
+#include <string>
+
+
+class logIn
+{
+private:
+  bool isEqualPass(const std::string i_password, const std::string i_repeatedPassword);
+  enum connectionMessage { wrongPassword, goodEntry, doesntExists, ocupatedUser};
```

Comment:
According with the coding standard, use C++11 scoped enumerations.
Also, it is recommended to place only one expression on each line.